site/docs-md/guides/community.md

```diff
@@ -18,23 +18,23 @@ contributors:
 
 [Capacitor Workflow for iOS and Android Apps &#8250;](https://www.youtube.com/watch?v=oXbRcpsytGQ)
 
-[Using the Capacitor in Vue.js app](https://akhromieiev.com/tutorials/using-capacitor-in-vue-js-app/)
+[Using the Capacitor in Vue.js app &#8250;](https://akhromieiev.com/tutorials/using-capacitor-in-vue-js-app/)
 
 ## APIs
 
-[Camera: Building an Ionic Framework Camera App](/docs/guides/ionic-framework-app)
+[Camera: Building an Ionic Framework Camera App &#8250;](/docs/guides/ionic-framework-app)
```

Comment:
I think the `&#8250;` is only intended for external links
```suggestion
[Camera: Building an Ionic Framework Camera App](/docs/guides/ionic-framework-app)
```

Reply:
OK